genie-web/src/main/java/com/netflix/genie/web/jpa/entities/aggregates/package-info.java

```diff
@@ -0,0 +1,28 @@
+/*
+ *
+ *  Copyright 2017 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+/**
+ * A package containing aggregates interfaces for Spring Data JPA.
+ *
+ * @author mprimi
+ * @since 4.0.0
+ */
+@ParametersAreNonnullByDefault
+package com.netflix.genie.web.jpa.entities.aggregates;
```

Comment:
Shouldn't this be in the previous commit as you already had a class in there within this package?

Reply:
Commit are sorted by date in GH web UI, but this commit comes first.